client/src/assets/blog/history_of_lead.ts

```diff
@@ -0,0 +1,45 @@
+import { BlogPost } from '@/assets/blog/blog_post';
+
+export const historyOfLead: BlogPost = {
```

Comment:
are we following up with images or not including them at all? https://docs.google.com/document/d/1F22E5OJIXAPCmVSa-Lf_rCdXEMCP9h_kYuGCTbE_Z_Y/edit?resourcekey=0-BCTyHAC_Zu-mkL0DSr3IMA#heading=h.ne0ydyij1p2c

Reply:
As discussed offline we'll add images separately just to unblock the content.